scripts/mbedtls_dev/crypto_knowledge.py

```diff
@@ -138,6 +138,9 @@ def is_public(self) -> bool:
         """Whether the key type is for public keys."""
         return self.name.endswith('_PUBLIC_KEY')
 
```

Comment:
Minor: please tweak line to `short_expression` to abbreviate `DH_FAMILY` like `ECC_FAMILY`.